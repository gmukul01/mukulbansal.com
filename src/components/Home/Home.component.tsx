import { Text } from '@medly-components/core';
import React from 'react';
import * as Styled from './Home.styled';

export const Home: React.SFC = React.memo(props => (
    <Styled.Home>
        <Text textSize="L1">Coming Soon</Text>
    </Styled.Home>
));
