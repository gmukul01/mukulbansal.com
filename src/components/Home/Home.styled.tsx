import { defaultTheme } from '@medly-components/theme';
import { centerAligned } from '@medly-components/utils';
import { styled } from '@styled';

export const Home = styled('div')`
    width: 100%;
    height: 100%;
    background-color: ${defaultTheme.colors.black};
    ${centerAligned('flex')}
`;
