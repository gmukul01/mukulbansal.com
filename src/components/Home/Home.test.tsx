import { render } from '@test-utils';
import React from 'react';
import { Home } from './Home.component';

describe('Home Component', () => {
    it('should render properly', () => {
        const { container } = render(<Home />);
        expect(container).toMatchSnapshot();
    });
});
