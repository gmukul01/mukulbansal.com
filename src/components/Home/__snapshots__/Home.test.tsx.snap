// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component should render properly 1`] = `
.c1 {
  font-size: 20px;
  font-weight: 400;
  color: inherit;
}

.c0 {
  width: 100%;
  height: 100%;
  background-color: #10161A;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div>
  <div
    class="c0"
  >
    <span
      class="c1"
    >
      Coming Soon
    </span>
  </div>
</div>
`;
